src/main/drivers/serial_uart.c

```diff
@@ -136,6 +140,9 @@ uartDeviceIdx_e uartDeviceIdxFromIdentifier(serialPortIdentifier_e identifier)
     // table is for UART only to save space (LPUART is handled separately)
 #define _R(id, dev) [id] = (dev) + 1
     static const uartDeviceIdx_e uartMap[] = {
+#ifdef USE_UART0
+        _R(SERIAL_PORT_UART0, UARTDEV_0),
```

Comment:
```suggestion
    _R(SERIAL_PORT_UART0, UARTDEV_0),
```

Reply:
This is aligned to `static const uartDeviceIdx_e .. {`